packages/rpc/test/clightning_rpc_test.dart

```diff
@@ -43,5 +43,26 @@ void main() {
       await client.simpleCall('delinvoice', params: params);
       client.close();
     });
+
+    test('Pay invoice method test', () async {
+      var client = RPCClient();
+      client.connect(rpcPath);
+      expect(client, isNotNull);
+      var params = <String, dynamic>{};
+      params['msatoshi'] = '100000msat';
+      params['label'] = 'from-dart';
+      params['description'] = 'This is a unit test';
+
+      var response = await client.simpleCall('invoice', params: params);
+      params = <String, dynamic>{};
+      params['bolt11'] = response["bolt11"];
+      await client.simpleCall('pay', params: params);
```

Comment:
```suggestion
     
      expect(() async =>  await client.simpleCall('pay', params: params), throwsA(TypeMatcher<LNClientException>()));
```

Reply:
I'm always worried when the test is green on the first run, which means the test is fake.

Let's fail and keep your eyes on the log failure!